examples/runner-multi-region/README.md

```diff
@@ -0,0 +1,86 @@
+# Example - Spot Runner - Multi-region
+
+In this scenario, we create multiple runner agents similarly to the _runner-public_ example but in different regions. Runners can be created with different configuration by instantiating the module multiple times. Runners will scale automatically based on configuration. The S3 cache can be shared cross runners by managing the cache outside the module.
+
+![runners-cache](https://github.com/npalm/assets/raw/master/images/terraform-aws-gitlab-runner/runner-cache.png)
+
+This examples shows:
+
+  - Multi-region deployment.
+  - Usages of public subnets.
+  - Usages of multiple runner instances sharing a common cache.
+  - Overrides for tag naming.
+  - Registration via GitLab token.
+  - Auto scaling using `docker+machine` executor.
+  - Register runner as [protected](https://docs.gitlab.com/ee/ci/runners/#protected-runners).
+
```

Comment:
I have suggest setting lock to project to false. The configuration is also locking to projected branches. I would eitehr remove this in the example or add it to this list.

Reply:
Not quite sure if I understand correctly, the configuration seems to enable protected runners via the *access_level* property 

```hcl
  # examples/runner-multi-region/main.tf

  gitlab_runner_registration_config = {
    registration_token = var.registration_token
    tag_list           = "docker_spot_runner"
    description        = "runner public - auto"
    locked_to_project  = "false"
    run_untagged       = "false"
    maximum_timeout    = "3600"
    access_level       = "ref_protected" # <--
  }
```

And the readme already mentions this this in the "This examples shows:" list. 
What should I add to the list?

Reply:
The link to the Gitlab documentation for protected runners seems to be outdated btw. I changed it here and in the *runner-public* example's readme.

Reply:
Alright, got it. I will remove the access level then since this is just copy & paste from the *runner-public* example anyway.